sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2493,6 +2505,10 @@ public void merge(final @NotNull ExternalOptions options) {
     if (options.isEnableBackpressureHandling() != null) {
       setEnableBackpressureHandling(options.isEnableBackpressureHandling());
     }
+
+    if (options.getCron() != null) {
+      setCron(options.getCron());
```

Comment:
`h` should we merge instead of replacing here? A user could set something in `OptionsConfiguration` on `Sentry.init` and something in an external config file and the settings from `Sentry.init` would get lost.